core/metacling/src/TCling.cxx

```diff
@@ -626,6 +626,16 @@ extern "C" int TCling__AutoLoadCallback(const char* className)
    return ((TCling*)gCling)->AutoLoad(className);
 }
 
+extern "C" int TCling__AutoLoadLibraries(std::string ModuleName)
+{
+   std::string filename = "lib" + ModuleName + ".so";
```

Comment:
What about Windows? IIRC we don't need the shared lib extension.

Reply:
What about module names that are not == library names? (E.g. Smatrix32 comes to my mind)